jerry-core/jerry.cpp

```diff
@@ -1318,6 +1318,17 @@ jerry_api_eval (const jerry_api_char_t *source_p, /**< source code */
 } /* jerry_api_eval */
 
 /**
+ * Perform GC
+ */
+void
```

Comment:
Shouldn't we enable the API only in `MEM_STATS` mode in the case (as, in general configuration, gc has no exact information on exact amount of bytes freed)?